test/adapter.test.ts

```diff
@@ -125,6 +125,13 @@ test('TestAdapter', async () => {
             ['bob', 'data2', 'write'],
             ['data2_admin', 'data2', 'read'],
             ['data2_admin', 'data2', 'write']]);
+
+        // Load Filtered Policy
+        /*await a.loadFilteredPolicy(e.getModel(), {
```

Comment:
@vinod827 Remove the comments. Run it on test.

Reply:
okay